packages/server/modules/acc/domain/constants.ts

```diff
@@ -0,0 +1,30 @@
+export const ImporterAutomateFunctions = {
+  svf2: {
+    functionId: '2909d29a9d',
+    functionReleaseId: '6bafd557e8'
+  }
+}
+
+export const AccSyncItemStatuses = {
```

Comment:
dont have to change this, but u can use StringEnum() and its corresponding type for less verbosity when it comes to defining these